action.yml

```diff
@@ -35,6 +35,21 @@ inputs:
     description: >
       Add sslmode to the connection string. Supported values are: "require", "verify-ca", "verify-full", "omit".
     default: 'require'
+  created_branch_annotation:
```

Comment:
the name suggests that the branch was already created when action was started.
Suggest to rename to just "annotation"

Reply:
Yes, already fixed.